### PR TITLE
chore: veANC - update voting escrow docs

### PR DESCRIPTION
**Pull request by jeury301**
## Summary of changes

- Updates docs for `voting_escrow` contract

## Report of required housekeeping

- [ ] Github issue OR spec proposal link
- [ ] Wrote tests
- [ ] Added a relevant changelog entry to CHANGELOG.md

----

## (FOR ADMIN) Before merging

- [ ] Added appropriate labels to PR
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" (coding standards)
- [ ] Confirm added tests are consistent with the intended behavior of changes
- [ ] Ensure all tests pass

### CODE REVIEW

**Comment by jeury301** [comment]
[veANC - Documentation](https://trello.com/c/6sarfzAm/209-veanc-documentation)

File: contracts/voting_escrow/README.md [hunk 1]
```diff
@@ -50,58 +54,26 @@ Withdraws whole amount of veANC if lock is expired.
   "withdraw": {}
```
Comment (cdocu11) [inline contracts/voting_escrow/README.md]: we added `amount` as a parameter of `withdraw`

Comment (jeury301) [reply contracts/voting_escrow/README.md]: @cdocu11 addressed!

File: contracts/voting_escrow/README.md [hunk 2]
```diff
@@ -9,99 +9,74 @@ gains 250 voting power. Voting power is linearly decreased by passed periods. On
...
 
 ```json
 {
-  "receive": {
-    "sender": "terra...",
-    "amount": "123",
-    "msg": "<base64_encoded_json_string>"
+  "extend_lock_amount": {
+    "user": "terra...",
+    "amount": 500
```
Comment (cdocu11) [inline contracts/voting_escrow/README.md]: 500 here should be "500" because the type of amount is Uint128.

**Comment by cdocu11** [comment]
The first paragraph says `Maximum lock time is 2 years which equals to 2.5 coefficient. For example, if the user locks 100 ANC for 2 years he
gains 250 voting power`, we should change the `2 years` into `4 years`.

File: contracts/voting_escrow/README.md (hunk 2 above)
Comment (jeury301) [reply contracts/voting_escrow/README.md]: addressed 👍

**Comment by jeury301** [comment]
> The first paragraph says `Maximum lock time is 2 years which equals to 2.5 coefficient. For example, if the user locks 100 ANC for 2 years he gains 250 voting power`, we should change the `2 years` into `4 years`.

addressed!

**Review by cdocu11 — APPROVED** [review]
LGTM

**Pull request merged**